packages/clipboard/src/index.ts

```diff
@@ -0,0 +1,72 @@
+import { Observable, from, of } from 'rxjs';
+import * as clipboardpoly from 'clipboard-polyfill';
+
+/**
+ * Clipboard
+ * This service is used to store/retrive a data from the system clipboard and
+ * if the system clipboard cannot be accessible, it will use the local storage
+ * to store and retrieve data.
+ *
+ * System clipboard provides ability to copy or paste the data from one browser window
+ * to other browser window.
+ * Local clipboard provides ability to copy or paste the data within the application.
+ */
+export class Clipboard {
+  /**
+   * This copies the given data to the system clipboard and as fallback,
+   * it copies to local storage.
+   * @param data data that needs to be copied to the clipboard
+   */
+  public copy(data: any): Observable<any> {
```

Comment:
This doesn't have to be an observable. Please use promises. We can get rid of `rxjs` from this module.

Reply:
Done.